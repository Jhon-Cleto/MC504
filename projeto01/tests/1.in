1 10
10 1
5342