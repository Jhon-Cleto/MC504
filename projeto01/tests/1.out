Multiplicação de Matrizes: A x B = C

Matriz A (1 x 10):
 004 003 009 003 001 002 001 008 005 000

Matriz B (10 x 1):
 001
 008
 004
 006
 005
 002
 007
 001
 006
 005

Matriz C (1 x 1):
 136
